bindings/jupyroot/python/JupyROOT/helpers/cppcompleter.py

```diff
@@ -44,23 +44,6 @@ class CppCompleter(object):
 
     >>> comp = CppCompleter()
     >>> comp.activate()
-    >>> for suggestion in comp._completeImpl("TH1"):
```

Comment:
What is that part of the test replaced with?

Reply:
Went in TTabComTests.

Reply:
That is similar but not equivalent.  I.e. This tested not only that it gave the right answer but also that the answer was accessible from jupyter.  So maybe replace it with a more stable choice.  eg.
```
>>> for suggestion in comp._completeImpl("TTre"):
>>> for suggestion in comp._completeImpl("TTreeF"):
```

Reply:
@etejedor gave green light but I will try adding `_completeImpl("TTreeF")` the other one has too many completion options :)